src/commands/set.js

```diff
@@ -25,12 +26,15 @@ const description = `Sets configuration <variable> to <value>. Variables availab
 	Examples:
 	- set json true
 	- set name my_custom_lisky
-	- set api.testnet true
+	- set api.network main
 `;
 
 const WRITE_FAIL_WARNING =
 	'Config file could not be written: your changes will not be persisted.';
 
+const NETHASH_ERROR_MESSAGE =
+	'Value must be a hex string with 64 characters. Alternatively, main, test and beta can be used.';
```

Comment:
Value must be a hex string with 64 characters, or one of `main`, `test` or `beta`.